### PR TITLE
add license check

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>
Add license check

### CODE REVIEW

File: __mocks__/styleMock.js [hunk 1]
```diff
@@ -1 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2018-2018 Red Hat, Inc.
```
Comment (akurinnoy) [inline __mocks__/styleMock.js]: 2020?

Comment (olexii4) [reply __mocks__/styleMock.js]: I got it from https://github.com/eclipse/che-dashboard.
 I can fix it)

**Review by sleshchenko — APPROVED** [review]
Please fix the year before merging

**Comment by akurinnoy** [comment]
@olexii4 why don't you use [eslint-plugin-header](https://www.npmjs.com/package/eslint-plugin-header) or similar tool?

File: __mocks__/styleMock.js (hunk 1 above)
Comment (olexii4) [reply __mocks__/styleMock.js]: I have done it

**Comment by olexii4** [comment]
@akurinnoy I just reuse the same approach which for my 'bump-deps' scrips. As for me, it is an easier way. I will be glad to see something better instead.

File: __mocks__/styleMock.js (hunk 1 above)
Comment (sleshchenko) [reply __mocks__/styleMock.js]: Maybe for that subproject `2020-2020` makes more sense than `2018-2020`?
@akurinnoy @olexii4 WDYT?

**Review by akurinnoy — APPROVED** [review]

File: __mocks__/styleMock.js (hunk 1 above)
Comment (olexii4) [reply __mocks__/styleMock.js]: https://github.com/che-incubator/che-dashboard-next/pull/25/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3
```
  "name": "che-dashboard",
  "version": "5.0.0",
  "description": "Dashboard for Eclipse CHE",
...
```
 '2018-2020' will be better in this case

**Pull request merged**